src/main/kotlin/com/devfloor/untitled/user/domain/User.kt

```diff
@@ -29,39 +30,39 @@ class User(
     github: String?,
     blog: String?,
     description: String?,
-) {
+) : BaseEntity() {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "user_id")
     var id: Long? = null
-        private set
+        protected set
 
     @Column(name = "name")
     var nickname: String = nickname
-        private set
+        protected set
 
     @Enumerated(value = EnumType.STRING)
     @Column(name = "type")
     var type: UserType = type
-        private set
+        protected set
```

Comment:
type이 교수인지 학생인지 그타입이지?
아예 val로 바꿔서 `protected set` 을 없애고 변경을 막아버리는건 어때 ?

```suggestion
    val type: UserType = type
```